protocol-models/src/main/resources/airbyte_protocol/v0/airbyte_protocol.yaml

```diff
@@ -464,6 +464,8 @@ definitions:
       - stream
       - sync_mode
       - destination_sync_mode
+      - current_generation_id
```

Comment:
It's `generation_id` below, not `current_generation_id`

Reply:
Update following the discussion in https://github.com/airbytehq/airbyte-protocol/pull/67#discussion_r1546892291